### DISCUSSION TITLE
HDS certification in France

### DISCUSSION THREAD

**Discussion by gvancassel** [supabase/supabase#30734] [Feature Requests]
Hi Supabase team, would you have any plans to have Supabase HDS certified to host / manage health data in France? Thanks!

**Comment by DanilSavine** [comment]
I would also be interested in the answer :)

**Comment by Alaa-Ben** [comment]
Interested as well !

**Comment by thomasbrodusch** [comment]
Interested aswell, can we have a clear answer about that crucial point please ?

**Comment by copostic** [comment]
Interested as well!

**Comment by etroadec** [comment]
Same here !

**Comment by Moona333** [comment]
Same here!
